### ISSUE TITLE
Split cell button for table doesnot work in mobile

### ISSUE THREAD

**Issue by Rash419** [CollaboraOnline/online#3962]
**Describe the bug**
Impress:
It doesnot have a Split Cell button in Table option

Writer:
It does have button but on clicking it doesnot do anything

**To Reproduce**
Steps to reproduce the behavior for writer:
1. Insert a table
2. try to spit the cell

**Expected behavior**
It should spit the cell

**Actual behavior**
doesnot work

**Comment by pedropintosilva** [comment]
True, I think this can be solved with the contextual toolbars https://github.com/CollaboraOnline/online/pull/3869

**Comment by pedropintosilva** [comment]
@Minion3665 could you please look into this. It seems quite old, is it still the case?

**Comment by Minion3665** [comment]
No, it's not still the case - although there is an issue with selecting stuff in the dialog still (see attached screen recording)

I think that should be filed in a separate ticket, though

https://github.com/user-attachments/assets/d249f813-72a3-4872-8dfd-228a5fcb2c40
